pkgs/top-level/php-packages.nix

```diff
@@ -46,15 +46,15 @@
 , fetchpatch
 }:
 
+let
+  builders = callPackages ../build-support/php/builders { };
```

Comment:
`callPackage`, on the other hand, should import a file that contains a function that returns a single derivation.

Reply:
Should I use `callPackage` here ?

Reply:
I would use plain `import`.

Reply:
Indeed. In addition to auto-wiring, the point of `callPackage{s,}` is to set up `.override`. That will not really work here anyway due to the nested structure.